skimage/metrics/set_metrics.py

```diff
@@ -0,0 +1,57 @@
+import numpy as np
+
+from ._set_metrics import hausdorff_distance_onesided
+
+
+def hausdorff_distance(image0, image1):
+    """
+    Calculate the Hausdorff distance between nonzero elements of given images.
```

Comment:
The docstring should be on the same line as the quotes (many UIs rely on this to render the first line as per PEP257).

```suggestion
    """Calculate the Hausdorff distance between nonzero elements of given images.
```

Reply:
Done